packages/api/test/e2e/api.calls.e2e.ts

```diff
@@ -36,13 +40,21 @@ describe('e2e api calls', () => {
     expect(block.header.hash.toString()).toEqual(blockHash.toString());
   });
 
-  it.skip('Query historical block', async () => {
-    // const blockNumber = 2500; //local
-    const blockNumber = 3759962; //Azalea
-    const bHash = await api.rpc.chain.getBlockHash(blockNumber);
-    const block = await api.rpc.chain.getBlock(bHash);
-    console.log('Signed block::', block.toHuman());
-    console.log('Signed block::', block.toJSON());
+  it('Query historical block from runtime version 36', async () => {
+    const blockNumber = 3759962; // old Azalea block at runtime version 36
+    const provider = 'wss://node-6711773975684325376.jm.onfinality.io/ws?apikey=57eebed1-6b87-4fcf-8bb7-3d285e3eabff';
```

Comment:
this is not good from a security standpoint (making the api key public). If there's no other way then I'm happy to merge this as it's an important fix

Reply:
I see what you mean.. I will check if I can have one node running from runtime version 36 with docker, may be... I will investigate

Reply:
Ideally we don't have to add a new docker image with each test, if we could keep the block data (e.g hex/json) in the test I think that would be more portable.
Otherwise some test where we upgrade a node's runtime, verify block, upgrade again, verify block, so on starting at v36 would be 🔥 and future proof